aspnetcore/blazor/tutorials/movie-database-app/part-1.md

```diff
@@ -0,0 +1,349 @@
+
+-->
+
+This article is the first part of the Blazor movie database app tutorial that teaches you the basics of building an ASP.NET Core Blazor Web App with features to manage a movie database.
+
+This part of the tutorial series covers how to create a Blazor Web App that adopts static server-side rendering (static SSR). Static SSR means that content is rendered on the server and sent to the client for display in response to individual requests.
+
+## Prerequisites
+
+:::zone pivot="vs"
+
+[Visual Studio (latest release)](https://visualstudio.microsoft.com/downloads/?utm_medium=microsoft&utm_source=learn.microsoft.com&utm_campaign=inline+link&utm_content=download+vs2022) with the **ASP.NET and web development** workload
+
+:::zone-end
+
+:::zone pivot="vsc"
+
+Latest releases of:
+
+* [Visual Studio Code](https://code.visualstudio.com/download)
+* [C# Dev Kit](https://marketplace.visualstudio.com/items?itemName=ms-dotnettools.csharp)
+* [.NET SDK](https://dotnet.microsoft.com/download/dotnet)
+
+The Visual Studio Code (VS Code) instructions for ASP.NET Core development in this tutorial use the [.NET CLI](/dotnet/core/tools/), which is part of the .NET SDK. .NET CLI commands are issued in VS Code's integrated [**Terminal**](https://code.visualstudio.com/docs/editor/integrated-terminal), which defaults to a [PowerShell command shell](/powershell/). The **Terminal** is opened by selecting **New Terminal** from the **Terminal** menu in the menu bar.
+
+:::zone-end
+
+:::zone pivot="cli"
+
+[.NET SDK (latest release)](https://dotnet.microsoft.com/download/dotnet)
+
+The [.NET CLI](/dotnet/core/tools/) is part of the .NET SDK. To issue commands that affect the project, open the command shell to the project's root folder.
+
+:::zone-end
+
+## Create a Blazor Web App
+
+:::zone pivot="vs"
+
+In Visual Studio:
+
+* Select **Create a new project** from the **Start Window** or select **File** > **New** > **Project** from the menu bar.
+* In the **Create a new project** dialog, select **Blazor Web App** from the list of project templates. Select the **Next** button.
+* In the **Configure your new project** dialog, name the project `BlazorWebAppMovies` in the **Project name** field, including matching the capitalization. Using this exact project name is important to ensure that the namespaces match for code that you copy from the tutorial into the app that you're building.
+* Confirm that the **Location** for the app is suitable. Set the **Place solution and project in the same directory** checkbox to match your preferred solution file location. Select the **Next** button.
+* In the **Additional information** dialog, use the following settings:
+  * **Framework**: Confirm that the [latest framework](https://dotnet.microsoft.com/download/dotnet) is selected. If Visual Studio's **Framework** dropdown list doesn't include the latest available .NET framework, [update Visual Studio](/visualstudio/install/update-visual-studio) and restart the tutorial.
```

Comment:
Shouldn't the framework version match the framework version for the doc?

Reply:
Yes ... I'll version this remark.